var/spack/repos/builtin/packages/py-itk/package.py

```diff
@@ -48,6 +49,38 @@ class PyItk(PythonPackage):
             sha256="e8dec75b4452bd2ee65beb4901b245fc3a2a2ccc46dfa008ae0b5b757718d458",
             expand=False,
         )
+
+        # version 5.3.0
+        version(
+            "5.3.0-cp37m",
```

Comment:
Let's leave the m off the version string

Reply:
All of the downloads listed for python-37 are listed in that way. I do not think it can just be dropped.
```
wget https://pypi.io/packages/cp37/i/itk/itk-5.3.0-cp37-cp37-manylinux_2_28_x86_64.whl
--2023-01-17 17:15:48--  https://pypi.io/packages/cp37/i/itk/itk-5.3.0-cp37-cp37-manylinux_2_28_x86_64.whl
Resolving pypi.io (pypi.io)... 2a04:4e42:400::223, 2a04:4e42:600::223, 2a04:4e42:200::223, ...
Connecting to pypi.io (pypi.io)|2a04:4e42:400::223|:443... failed: Connection refused.
Connecting to pypi.io (pypi.io)|2a04:4e42:600::223|:443... failed: Connection refused.
Connecting to pypi.io (pypi.io)|2a04:4e42:200::223|:443... failed: Connection refused.
Connecting to pypi.io (pypi.io)|2a04:4e42::223|:443... failed: Connection refused.
Connecting to pypi.io (pypi.io)|151.101.64.223|:443... connected.
HTTP request sent, awaiting response... 301 Redirect to Primary Domain
Location: https://pypi.org/packages/cp37/i/itk/itk-5.3.0-cp37-cp37-manylinux_2_28_x86_64.whl [following]
--2023-01-17 17:15:49--  https://pypi.org/packages/cp37/i/itk/itk-5.3.0-cp37-cp37-manylinux_2_28_x86_64.whl
Resolving pypi.org (pypi.org)... 2a04:4e42:400::223, 2a04:4e42::223, 2a04:4e42:600::223, ...
Connecting to pypi.org (pypi.org)|2a04:4e42:400::223|:443... failed: Connection refused.
Connecting to pypi.org (pypi.org)|2a04:4e42::223|:443... failed: Connection refused.
Connecting to pypi.org (pypi.org)|2a04:4e42:600::223|:443... failed: Connection refused.
Connecting to pypi.org (pypi.org)|2a04:4e42:200::223|:443... failed: Connection refused.
Connecting to pypi.org (pypi.org)|151.101.128.223|:443... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://files.pythonhosted.org/packages/cp37/i/itk/itk-5.3.0-cp37-cp37-manylinux_2_28_x86_64.whl [following]
--2023-01-17 17:15:49--  https://files.pythonhosted.org/packages/cp37/i/itk/itk-5.3.0-cp37-cp37-manylinux_2_28_x86_64.whl
Resolving files.pythonhosted.org (files.pythonhosted.org)... 2a04:4e42:84::319, 146.75.81.63
Connecting to files.pythonhosted.org (files.pythonhosted.org)|2a04:4e42:84::319|:443... failed: Connection refused.
Connecting to files.pythonhosted.org (files.pythonhosted.org)|146.75.81.63|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-01-17 17:15:49 ERROR 404: Not Found.
```

Reply:
It can't be dropped from the URL, but it can be dropped from the version string. See our existing cp37 versions.

Reply:
Ah, got it.